spec/mongoid/touchable_spec_models.rb

```diff
@@ -6,15 +6,15 @@ class Building
       include Mongoid::Document
       include Mongoid::Timestamps
 
-      embeds_many :entrances
-      embeds_many :floors
+      embeds_many :entrances, class_name: "TouchableSpec::Embedded::Entrance"
+      embeds_many :floors, class_name: "TouchableSpec::Embedded::Floor"
```

Comment:
Is this needed because Mongoid does not resolve class names correctly?

Reply:
I believe so